### PR TITLE
core: Make rsc error messages a bit more verbose

### PR DESCRIPTION
**Pull request by ddeboer**
### Description

Ref #1167.

### Checklist

- [ ] documentation updated
- [ ] tests added
- [ ] no BC breaks

### CODE REVIEW

**Comment by mworrell** [comment]
Is this already done in the master?

File: src/models/m_rsc_update.erl [hunk 1]
```diff
@@ -571,23 +572,23 @@ preflight_check(Id, [{name, Name}|T], Context) when Name =/= undefined ->
...
+            throw({error, {duplicate_page_path, Path}})
     end;
 preflight_check(Id, [{uri, Uri}|T], Context) when Uri =/= undefined ->
     case z_db:q1("select count(*) from rsc where uri = $1 and id <> $2", [Uri, Id], Context) of
         0 ->
             preflight_check(Id, T, Context);
         _N ->
             lager:warning("Trying to insert duplicate uri ~p", [Uri]),
-            throw({error, duplicate_uri})
+            throw({error, duplicate_uri, Uri})
```
Comment (ddeboer) [inline src/models/m_rsc_update.erl]: Must be 3-tuple

**Comment by mworrell** [comment]
What shall we do with this MR?

In master we did away with the throw-statements and just return an error tuple.

**Comment by mworrell** [comment]
Closing this, as it won't be directly mergeable in the 1.0 and we keep the 0.x errors as-is for now.

**Pull request closed without merging**